### PR TITLE
Feature/skeleton

### PR DESCRIPTION
**Pull request by iimpulse**
Creating a python package skeleton for phenotron.

Adding omop2obo and phenopackets as dependencies :)

### CODE REVIEW

**Review by pnrobinson — APPROVED** [review]
Looks great

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+=======
+Credits
+=======
+
+Contributors
```
Comment (callahantiff) [inline AUTHORS.rst:5]: Should Peter also be added here?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+------------
+
+Ready to contribute? Here's how to set up `phenotron` for local development.
+
+1. Fork the `phenotron` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/phenotron.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
```
Comment (callahantiff) [inline CONTRIBUTING.rst:67]: Extra colon at the end of this line 😄

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    $ mkvirtualenv phenotron
+    $ cd phenotron/
+    $ python setup.py develop
+
+4. Create a branch for local development::
```
Comment (callahantiff) [inline CONTRIBUTING.rst:73]: Extra colon here too 🗡️

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+
+    $ python -m unittest tests.test_phenotron
+
+Deploying
+---------
+
+A reminder for the maintainers on how to deploy.
+Make sure all your changes are committed (including an entry in HISTORY.rst).
+Then run::
```
Comment (callahantiff) [inline CONTRIBUTING.rst:122]: same here 🍡

File: requirements_dev.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+wheel
+watchdog
+flake8
+tox
+coverage
+Sphinx
+twine
+Click
+black
+omop2obo==1.0.0
```
Comment (callahantiff) [inline requirements_dev.txt]: any reason to set this? I suspect I will be pushing new releases soon that will extend the functionality of v1.0.0

**Review by callahantiff — COMMENTED** [review]
Great work! Found 3 tiny errors in the contributing and then asked a question about the version of OMOP2OBO that was set :D

File: CONTRIBUTING.rst (hunk 2 above)
Comment (iimpulse) [reply CONTRIBUTING.rst:67]: in rst this is how you make the code "wells"

**Pull request merged**